src/App.jsx

```diff
@@ -1,14 +1,46 @@
+import { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
+import messages from './data/messages.json'
+
+
+// import data from messages.json
+const LOG = messages;
```

Comment:
You can directly name `messages` as `LOG` on line 4. 
```suggestion
import LOG from './data/messages.json';
```

Be sure to terminate a statement with semi-colons.